src/go/util/request.go

```diff
@@ -18,39 +18,58 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"time"
 
+	"github.com/golang/glog"
 	"github.com/golang/protobuf/proto"
 )
 
-func callWithAccessToken(client *http.Client, path, method, token string) ([]byte, error) {
+var Retries = 30
```

Comment:
Can we implement as generic OnStatusRetry.  At first, status list can be just 429,  later, we can add other status. 

Config may look like this map:
OnStatusRetryConfig:
{
   [429, 433] => {RetryNum: 30,  RetryInterval: 3}
   [500] =>   {RetryNum: 5,  RetryInterval: 1}
}

Reply:
Hmm.  may need to have different retry policy with different host,  so the config map can be

host ->  []status -> {retry_num, retry_interval}

Reply:
Done.